### PR TITLE
feat!: transfer gcs action

### PR DESCRIPTION
**Pull request by bharathkkb**
- Transfer action from https://github.com/google-github-actions/setup-gcloud/tree/master/upload-cloud-storage

### CODE REVIEW

File: .github/workflows/upload-cloud-storage-it.yml [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+    name: with setup-gcloud
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - id: build
+        name: Build dist
+        run: |-
+          npm install
+          npm run build
+      - uses: GoogleCloudPlatform/github-actions/setup-gcloud@master
```
Comment (averikitsch) [inline .github/workflows/upload-cloud-storage-it.yml:16]: update

File: README.md [hunk 2]
```diff
@@ -1 +1,195 @@
...
+
+There are a few ways to authenticate this action. The caller must have
+permissions to access the secrets being requested.
+
+### Via the setup-gcloud action
+
+You can provide credentials using the [setup-gcloud][setup-gcloud] action:
+
+```yaml
+- uses: GoogleCloudPlatform/github-actions/setup-gcloud@master
```
Comment (averikitsch) [inline README.md:149]: update

File: action.yml [hunk 3]
```diff
@@ -0,0 +1,26 @@
+name: Cloud Storage Uploader
+description: Upload files or folders to GCS buckets
+author: GoogleCloudPlatform
```
Comment (averikitsch) [inline action.yml:3]: nit: Google LLC

**Review by averikitsch — APPROVED** [review]

File: .github/workflows/upload-cloud-storage-it.yml (hunk 1 above)
Comment (bharathkkb) [reply .github/workflows/upload-cloud-storage-it.yml]: done

File: README.md (hunk 2 above)
Comment (bharathkkb) [reply README.md]: done

File: action.yml (hunk 3 above)
Comment (bharathkkb) [reply action.yml]: done

**Pull request merged**